File: cpp/412. Fizz Buzz.cpp
class Solution {
public:
    vector<string> fizzBuzz(int n) {
        
        vector<string> res;
        
        for(int i = 1; i <= n; i++){
            
            if(i%3 == 0 && i%5 == 0){
                res.push_back("FizzBuzz");
            } else if(i%3 == 0){
                res.push_back("Fizz");
            } else if(i%5 == 0){
                res.push_back("Buzz");
            } else {
                res.push_back(to_string(i));
            }
            
        }
        
        return res;
        
    }
};

File: py/412. FizzBuzz.py
class Solution:
    def fizzBuzz(self, n: int) -> List[str]:
        l=[]
        for i in range(1,n+1):
            if i%3==0 and i%5==0:
                l.append("FizzBuzz")
            elif i%3==0:
                l.append("Fizz")
            elif i%5==0:
                l.append("Buzz")
            else:
                l.append(str(i))
        return l
